locale/shine/extensions/mapvote/csCS.json

```diff
@@ -44,8 +44,8 @@
 	"NOMINATED_MAP": "{TargetName} nominoval {MapName} k hlasování o mapu.",
 	"RTV_DISABLED": "RTV bylo vypnuto.",
 	"RTV_VOTED": "{TargetName} hlasoval pro změnu mapy (potřeba {VotesNeeded:Pluralise:další|dalších} {VotesNeeded} {VotesNeeded:Pluralise:hlas|hlasů}).",
-	"PLAYER_VOTED": "{TargetName} {Revote:BoolToPhrase:znovu hlasoval|hlasoval} pro {MapName} ({Votes} {Votes:Pluralise:hlas|hlasů} pro tuto mapu, {TotalVotes} celkem)",
-	"PLAYER_VOTED_PRIVATE": "{Revote:BoolToPhrase:Znovu jsi hlasoval|Hlasoval jsi} pro {MapName} ({Votes} {Votes:Pluralise:hlas|hlasů} pro tuto mapu, {TotalVotes} celkem)",
+	"PLAYER_VOTED": "{TargetName} {Revote:BoolToPhrase:změnil hlas na|hlasoval pro} {MapName} (hlasů pro tuto mapu: {Votes}, celkem: {TotalVotes})",
+	"PLAYER_VOTED_PRIVATE": "{Revote:BoolToPhrase:Změnil jsi hlas na|Hlasoval jsi pro} {MapName} (hlasů pro tuto mapu: {Votes}, celkem: {TotalVotes})",
```

Comment:
Does the original English phrasing not translate well to Czech, or have you changed these two because you think it looks better like this?

If it's the latter, then that's a little out of scope for a translation. Each one should have consistent phrasing with the original English unless it really doesn't translate.